splipy/IO/svg.py

```diff
@@ -7,6 +7,15 @@
 import re
 from .master import MasterIO
 
+def read_number_and_unit(mystring):
+    try:
+        for i in range(1, len(mystring)):
+            number=float(mystring[:i])
+    except ValueError:
+        unit=mystring[i-1:]
+    return (number, unit)
```

Comment:
I see you subscribe to the brute force school of problem solving. :smile:

Reply:
If it works it ain't broke 😉  I found that any clever regexp nastyness is always going to fail for wierd float formatting such as [+-.,e] or similar, so being a non-performance-critical part of the code, I thought this was nice enough 😆